plugins/registry/handler.go

```diff
@@ -199,13 +211,26 @@ func (rh *registryHandler) relay(ctx context.Context, p registryHandlerParams) {
 	}
 
 	if req.Method == http.MethodPut && resourceType == "manifests" {
-		logger.Info("tag:", resourceID)
-		logger.Info("model ID:", matches[1])
-		// DIGEST AT dockerContentDigest := resp.Header.Get("Docker-Content-Digest")
-		// CREATION TIME createdAt := time.Now()
+		// TODO Call create tag endpoint in artifact-backend.
+
+		// Deploy model.The previous operations are idempotent so it should be
+		// safe to repeat them if we fail here.
+		//
+		// TODO in the future the registry will handle more than model images,
+		// so this operation won't always be necessary. A much better pattern
+		// is publishing the push operation success as an event and let the
+		// clients to consume and act upon it (artifact to register the tag
+		// creation time, model to deploy the image...).
+
+		// TODO call UserModel or OrganizationModel depending on username == namespace.
+		modelName := fmt.Sprintf("users/%s/models/%s:%s", namespace, repository, resourceID)
```

Comment:
@heiruwu @tonywang10101 here the model name would be something like:

```
users/jvalles/models/jvalles/llava-34b:latest
```

❓ Is that correct?